packages/calypso-e2e/src/lib/components/site-select-component.ts

```diff
@@ -68,4 +56,16 @@ export class SiteSelectComponent {
 		// Assert the resulting URL is in the form of <protocol><calypsoBaseURL>/home/<url>.
 		await this.page.waitForURL( `**/home/${ url }` );
 	}
+
+	/**
+	 * Clicks on the Add Site button at bottom of the site selector within the sidebar.
+	 *
+	 * @returns {Promise<void>} No return value.
+	 */
+	async addSite(): Promise< void > {
+		await Promise.all( [
+			this.page.waitForNavigation(),
+			this.page.click( 'a:text("Add New Site")' ),
+		] );
```

Comment:
I debated where to place this method, but settled on the `SiteSelectComponen` since the class name makes it pretty clear the component that is rendered on screen when `Switch Site` is clicked is that of the `SiteSelectComponent`.

Reply:
Hmm, this one definitely is tricky. I don't want to start too much back-and-forth, but I will say that this does feel a little weird to me to be in the SiteSelectComponent. It feels like it should either be its own new component POM, or part of the sidebar.

To me, we seem to define POMs not based on overall functional intention, but actual physical structure in the UI. From that perspective, this is pretty separate from the whole-page site selector component.

Reply:
I don't think this deserves its own component/page, since it isn't physically or functionally separate from either the Sidebar or SiteSelect. Bit of a stretch, but an argument could be that 'creating a new site will switch the user to that site!'. 

With that said, I am with you that it feels...a bit off. If anything I'd like to see this placed on the SidebarComponent. What do you think?

Reply:
I've moved it to the SidebarComponent in 6ee92e2dbe94c57cdec6f968e1423e98211f8841.